.github/workflows/deno.yml

```diff
@@ -12,7 +12,7 @@ on:
   push:
     branches: ["main"]
   pull_request:
-    branches: ["main"]
+    branches: ["main", "enter"]
```

Comment:
[match *](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters)